config/initializers/puppet.rb

```diff
@@ -12,7 +12,6 @@
     :priority => 100
 end
 
-FactImporter.register_fact_importer :puppet, PuppetFactImporter, true
```

Comment:
Where do you plan to register `PuppetFactImporter`, I couldn't find it in the PR?